addons/scroll-project-titles/script.js

```diff
@@ -0,0 +1,7 @@
+export default async function ({ addon, console, msg }) {
+  addon.self.addEventListener("disabled", function() {
+    if (document.querySelector(".project-title.no-edit")) {
```

Comment:
This check is probably unnecessary, especially since it only runs when dynamically disabling the addon.

Reply:
It can still prevent errors, and it also makes sure that it doesn't affect titles that you can edit. It's important to reset the scroll, that way you don't get all the project titles stuck halfway through scrolling without any way of resetting it.